AudibleApi/Api [partial].cs

```diff
@@ -33,21 +33,21 @@ public Task<HttpResponseMessage> AdHocAuthenticatedGetAsync(string requestUri)
 		public Task<HttpResponseMessage> AdHocAuthenticatedGetAsync(string requestUri, IHttpClientActions client)
 			=> AdHocAuthenticatedRequestAsync(requestUri, HttpMethod.Get, client);
 
-		public async Task<HttpResponseMessage> AdHocAuthenticatedRequestAsync(string requestUri, HttpMethod method, IHttpClientActions client, JObject postData = null)
+		public async Task<HttpResponseMessage> AdHocAuthenticatedRequestAsync(string requestUri, HttpMethod method, IHttpClientActions client, JObject requestBody = null)
 		{
 			if (requestUri is null)
 				throw new ArgumentNullException(nameof(requestUri));
 			if (string.IsNullOrWhiteSpace(requestUri))
 				throw new ArgumentException($"{nameof(requestUri)} may not be blank");
 			if (method is null)
 				throw new ArgumentNullException(nameof(method));
-			if (method.Method == HttpMethod.Post.Method && postData is null)
-				throw new ArgumentNullException(nameof(postData), $"Must provide post data when using {nameof(HttpMethod)}.{nameof(HttpMethod.Post)}");
+			if (requestBody is null && (method.Method == HttpMethod.Post.Method || method.Method == HttpMethod.Put.Method))
```

Comment:
I don't remember where I picked up this trick which I later baked into Dinah.Core: I made the `.In<T>()` ext. method to condense `x == a || x == b || x == c ...` . By using `params`, I get an array for free, on which I can call `Contains`. It's just syntactic sugar. I like the option since it's sometimes more expressive.

Reply:
Convenient! I'll remember that one.